azurerm/resource_arm_app_service_certificate_test.go

```diff
@@ -0,0 +1,227 @@
+					resource.TestCheckResourceAttr(resourceName, "thumbprint", "7B985BF42467791F23E52B364A3E8DEBAB9C606E"),
+				),
+			},
+			{
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"pfx_blob", "password"},
+			},
+		},
+	})
+}
+
+func TestAccAzureRMAppServiceCertificate_PfxNoPassword(t *testing.T) {
+	resourceName := "azurerm_app_service_certificate.test"
+	ri := tf.AccRandTimeInt()
+	location := testLocation()
+
+	config := testAccAzureRMAppServiceCertificatePfxNoPassword(ri, location)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMAppServiceCertificateDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "thumbprint", "7B985BF42467791F23E52B364A3E8DEBAB9C606E"),
+				),
+			},
+			{
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"pfx_blob"},
+			},
+		},
+	})
+}
+
+func TestAccAzureRMAppServiceCertificate_KeyVault(t *testing.T) {
+	resourceName := "azurerm_app_service_certificate.test"
+	ri := tf.AccRandTimeInt()
+	location := testLocation()
+
+	config := testAccAzureRMAppServiceCertificateKeyVault(ri, location)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMAppServiceCertificateDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "thumbprint", "7B985BF42467791F23E52B364A3E8DEBAB9C606E"),
+				),
+			},
+			{
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"key_vault_secret_id"},
+			},
+		},
+	})
+}
+
+func testAccAzureRMAppServiceCertificatePfx(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestwebcert%d"
+  location = "%s"
+}
+
+resource "azurerm_app_service_certificate" "test" {
+  name = "acctest%d"
+  resource_group_name = azurerm_resource_group.test.name
+  location = azurerm_resource_group.test.location
+  pfx_blob = filebase64("testdata/app_service_certificate.pfx")
+  password = "terraform"
+}
+`, rInt, location, rInt)
+}
+
+func testAccAzureRMAppServiceCertificatePfxNoPassword(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestwebcert%d"
+  location = "%s"
+}
+
+resource "azurerm_app_service_certificate" "test" {
+  name = "acctest%d"
+  resource_group_name = azurerm_resource_group.test.name
```

Comment:
And here

Reply:
Sorry, will make sure `fmt` is run on the test configurations next time. Perhaps `terrafmt` can be used here.